hphp/runtime/vm/jit/smashable-instr-ppc64.cpp

```diff
@@ -73,74 +73,59 @@ TCA emitSmashableCall(CodeBlock& cb, CGMeta& fixups, TCA target,
 }
 
 TCA emitSmashableJmp(CodeBlock& cb, CGMeta& fixups, TCA target) {
-  return EMIT_BODY(cb, fixups, branchFar, target);
+  return EMIT_BODY(cb, fixups, branchFar, target, ppc64_asm::BranchConditions::Always,
+                   ppc64_asm::LinkReg::DoNotTouch, ppc64_asm::ImmType::TocOnly, true);
```

Comment:
Our linter requires all lines to be <= 80 chars.  Please reflow all over-long lines in this file (and others you've touched).

Reply:
Done.